docker/package/tezos_voting_wizard.py

```diff
@@ -202,6 +195,7 @@ def search_client_config(self, field, default):
     def collect_baking_info(self):
```

Comment:
Likewise IMO when no baking is detected the message here:
```
With no tezos-baking.service running, this wizard will use
the default directory for this network: {network_dir}
```
is unnecessary, because it's immediately followed by the recap from `check_data_correctness` which contains a line for the data directory,

This info is hardly ever necessary to know, IMO there's no need to repeat it twice.

Reply:
I disagree. With no message like this, there is no indication that it's not possible to change the directory like it's possible to change the other two parameters shown: node RPC address and the voter key. It's not a long message, so cutting it would not really conserve screen space, but IMO would make it (a bit) less clear how the wizard operates.

Reply:
:thinking: okay, I'm not sure if that would be a problem, but it's a fair point.

However isn't this the case also when there is a baking service running? IIRC there is no such message in that case tho.

Reply:
That's true, but from the docs I think it should be more-or-less clear that the two wizards are able to somewhat work in tandem. I'm not against adding a line like "With the service running, it will use the directory specified in the service config" (though I don't think it's strictly necessary), but I'm against removing this one.

Reply:
Ok, I suppose that's fair. 

I don't mind these lines so much, so I'm ok with leaving them in :ok_hand: